mllib/src/main/scala/org/apache/spark/mllib/tree/DecisionTree.scala

```diff
@@ -38,8 +38,9 @@ import org.apache.spark.util.random.XORShiftRandom
 /**
  * A class which implements a decision tree learning algorithm for classification and regression.
  * It supports both continuous and categorical features.
+ *
  * @param strategy The configuration parameters for the tree algorithm which specify the type
- *                 of algorithm (classification, regression, etc.), feature type (continuous,
+ *                 of decision tree (classification or regression), feature type (continuous,
  *                 categorical), depth of the tree, quantile calculation strategy, etc.
```

Comment:
`Strategy` includes the `categoricalFeaturesInfo` which I think is where this part of the comment comes from.

It actually could be a little confusing, but I don't think it's a major issue. It could read something like `which features are categorical` which would be a little more accurate.